.github/scripts/check_owners.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script will check for an OWNERS file in all task and pipeline 
+# directories provided either via DIRECTORIES env var, or as 
+# arguments when running the script.
+#
+# Requirements:
+# - A GitHub token with permission to read members of a team in
+#   an organisation.
+#
+# Examples of usage:
+# export DIRECTORIES="mydir/tasks/apply-mapping some/other/dir"
+# ./check_owners.sh
+#
+# or
+#
+# ./check_owners.sh mydir/tasks/apply-mapping some/other/dir
+
+if [ $# -gt 0 ]; then
+  DIRECTORIES=$@
+fi
+
+if [ -z "${DIRECTORIES}" ]; then
+  echo Error: No directories as argument.
+  echo Usage:
+  echo "$0 [item1] [item2] [...]"
+  exit 1
+fi
+
+# check every item is a directory
+for DIR in $DIRECTORIES; do
+  if [[ -d "$DIR" ]]; then
+    true
+  else
+    echo "Error: Not a directory: $DIR"
+    exit 1
+  fi
+done
+
+for DIR in $DIRECTORIES
+do
+
+  SHORT_DIR=$(echo $DIR | cut -d '/' -f -2)
+  OWNERS=${SHORT_DIR}/OWNERS
+
+  if [ ! -f $OWNERS ]; then
+    echo Error: OWNERS file does not exist: $SHORT_DIR
+    exit 1
+  fi
+
+done
+
+TEAM_NAME=release-service-maintainers
+
+for DIR in $DIRECTORIES
```

Comment:
Why do `for DIR in $DIRECTORIES` twice and declare `SHORT_DIR` and `OWNERS` twice? Couldn't you just do the REGEX check in the first loop, after line 49?